released/packages/coq-vcfloat/coq-vcfloat.2.1/opam

```diff
@@ -0,0 +1,43 @@
+opam-version: "2.0"
+version: "2.1"
+synopsis: "VCFloat: Floating Point Round-off Error Analysis"
+description: "VCFloat is a tool for Coq proofs about floating-point round-off error."
+authors: [
+  "Andrew W. Appel"
+  "Ariel E. Kellison"
+  "Tahina Ramananandro"
+  "Paul Mountcastle"
+  "Benoit Meister"
+  "Richard Lethin"
+]
+homepage: "https://verinum.org/vcfloat/"
+maintainer: "Andrew W. Appel <appel@princeton.edu>"
+dev-repo: "git+https://github.com/VeriNum/vcfloat"
+bug-reports: "https://github.com/VeriNum/vcfloat/issues"
+license: "LGPL-3.0-or-later"
+
+build: [
+  [ make "-C" "vcfloat" "depend" ]
+  [ make "-C" "vcfloat" "-j%{jobs}%" "vcfloat2"]
+]
+install: [
+  [make "-C" "vcfloat" "-j%{jobs}%" "install" "INSTALLDIR=%{lib}%/coq/user-contrib/vcfloat"]
+]
+run-test: [
+  [make "-C" "vcfloat" "-j%{jobs}%" "tests"]
+]
+depends: [
+  "coq" {>= "8.16" & < "8.18~"}
+  "coq-flocq" {>= "4.1.0" & < "5.0"}
```

Comment:
```suggestion
  "coq-flocq" {>= "4.1.0" & < "5.0"}
  "coq-coquelicot"
```
I see the following in the CI log:
```
# coqc -Q . vcfloat IEEE754_extra.v
# File "./RAux.v", line 37, characters 0-42:
# Error: Cannot find a physical path bound to logical path
# Coquelicot with prefix Coquelicot.
```
Do you maybe have a bound on least version of Coquelicot as well?

Reply:
The dependency on Coquelicot is unnecessary.  I'll commit a new version of the vcfloat without it, then I'll adjust this P.R.

Reply:
Done.  I hope nobody is offended that I moved the tag 2.1 on the VeriNum/vcfloat repository to a new commit, but now this PR to opam-coq-archive should build without any dependency on coquelicot.